mopidy/http/actor.py

```diff
@@ -179,8 +185,29 @@ def _get_static_request_handlers(self):
     def _get_mopidy_request_handlers(self):
         return [
             (
-                "/",
+                r"/",
                 tornado.web.RedirectHandler,
                 {"url": "/mopidy/", "permanent": False},
             )
         ]
+
+    def _get_cookie_secret(self):
```

Comment:
Given Python 3 and a couple of simplifications, I suggest:

``` python
def _get_cookie_secret(self):
    secret_path = Extension.get_data_dir(self.config) / "cookie_secret"
    if not secret_path.is_file():
        cookie_secret = secrets.token_hex(32)
        secret_path.write_text(cookie_secret)
    else:
        cookie_secret = secret_path.read_text().strip()
        if not cookie_secret:
            logging.error(f"HTTP server could not find cookie secret in {secret_path}")
    return cookie_secret
```

Reply:
done.
codecov is not passing, can you help me understanding why ? Because i don't have a test for _get_cookie_secret ?
Thank you !

Reply:
Well, wasn't that hard ! 
We have a test for _get_secure_cookie (testing that we are able to generate a cookie)
and a small login procedure test